Error: Cannot find module 'cli-ux' (looks like cli-ux is not saved in the package.json)

**Describe the issue**
On installation using npm i -g, ghs is missing the cli-ux package (looks like it wasn't saved in the package.json)

Version: @anomalyhq/github-secrets-cli/1.0.1 darwin-x64 node-v14.13.0

**Steps To Reproduce**
1. Install using `npm install -g @anomaly/github-secrets-cli`
2. Check that ghs is installed using `ghs -v`, see that it's correctly installed
3. Try `ghs config:set`
4. Returns:

````` 
     Error: Cannot find module 'cli-ux' ...
    Require stack:
    - /usr/local/lib/node_modules/@anomalyhq/github-secrets-cli/lib/commands/config/set.js
    - /usr/local/lib/node_modules/@anomalyhq/github-secrets-cli/node_modules/@oclif/config/lib/plugin.js
    - /usr/local/lib/node_modules/@anomalyhq/github-secrets-cli/node_modules/@oclif/config/lib/config.js
    - /usr/local/lib/node_modules/@anomalyhq/github-secrets-cli/node_modules/@oclif/config/lib/index.js
    - /usr/local/lib/node_modules/@anomalyhq/github-secrets-cli/node_modules/@oclif/command/lib/command.js
    - /usr/local/lib/node_modules/@anomalyhq/github-secrets-cli/node_modules/@oclif/command/lib/index.js
    - /usr/local/lib/node_modules/@anomalyhq/github-secrets-cli/bin/run
    Code: MODULE_NOT_FOUND
`````

**Expected behavior**
Should prompt for organization: `Which organisation does the repo belong to?: `. This behavior works as expected after installing cli-ux in the ghs package directory.

**Logs**
See Above!

--- a/package.json
+++ b/package.json
@@ -13,6 +13,7 @@
     "@oclif/errors": "^1",
     "@oclif/plugin-help": "^3",
     "@octokit/request": "^5.4.5",
+    "cli-ux": "^5.5.0",
     "fs-extra": "^9.0.1",
     "node-emoji": "^1.10.0",
     "tslib": "^1",
@@ -33,6 +34,7 @@
     "globby": "^10",
     "mocha": "^5",
     "nyc": "^14",
+    "prettier": "^2.1.2",
     "ts-node": "^8",
     "typescript": "^3.3"
   },
@@ -72,7 +74,8 @@
     "prepack": "rm -rf lib && tsc -b && oclif-dev manifest && oclif-dev readme",
     "test": "nyc --extension .ts mocha --forbid-only \"test/**/*.test.ts\"",
     "version": "oclif-dev readme && git add README.md",
-    "prepare": "rm -rf lib && tsc -b"
+    "prepare": "rm -rf lib && tsc -b",
+    "prettier": "prettier --write './src/**/*.{js,ts}'"
   },
   "types": "lib/index.d.ts"
 }

--- a/src/commands/config/get.ts
+++ b/src/commands/config/get.ts
@@ -1,6 +1,6 @@
-import { Command } from '@oclif/command'
-import { CLIError } from '@oclif/errors'
-import { configuration } from '../../utils/config'
+import {Command} from '@oclif/command'
+import {CLIError} from '@oclif/errors'
+import {configuration} from '../../utils/config'
 
 export default class ConfigGet extends Command {
   static description = 'Outputs your configuration.'

--- a/src/commands/config/set.ts
+++ b/src/commands/config/set.ts
@@ -1,12 +1,12 @@
-import { Command, flags } from '@oclif/command'
-import { CLIError } from '@oclif/errors'
+import {Command, flags} from '@oclif/command'
+import {CLIError} from '@oclif/errors'
 import cli from 'cli-ux'
 import fs from 'fs-extra'
 import emoji from 'node-emoji'
 import path from 'path'
 
 export default class ConfigSet extends Command {
-  static description = 'Update you configuration'
+  static description = 'Update your configuration'
 
   static flags = {
     org: flags.string({
@@ -27,7 +27,7 @@ export default class ConfigSet extends Command {
   }
 
   async run() {
-    const { flags } = this.parse(ConfigSet)
+    const {flags} = this.parse(ConfigSet)
     const config = path.join(this.config.configDir, 'config.json')
 
     try {
@@ -38,7 +38,7 @@ export default class ConfigSet extends Command {
         flags.repo = await cli.prompt('What is the name of the repo?')
         flags.personalAccessToken = await cli.prompt(
           'What is your GitHub Personal Access Token?',
-          { type: 'hide' }
+          {type: 'hide'}
         )
       }
 
@@ -53,7 +53,7 @@ export default class ConfigSet extends Command {
         'Your GHS CLI configuration has been generated!',
         emoji.get('rocket')
       )
-      this.exit()
+      // this.exit(0)
     } catch (error) {
       this.error(new CLIError(error) || 'A GHS CLI error has occurred.', {
         exit: 1,

--- a/src/commands/secrets/get.ts
+++ b/src/commands/secrets/get.ts
@@ -1,13 +1,13 @@
-import { Command, flags } from '@oclif/command'
-import { CLIError } from '@oclif/errors'
-import { request } from '@octokit/request'
-import { configuration } from '../../utils/config'
+import {Command, flags} from '@oclif/command'
+import {CLIError} from '@oclif/errors'
+import {request} from '@octokit/request'
+import {configuration} from '../../utils/config'
 
 export default class SecretsGet extends Command {
-  static description = 'Fetch a list os set secrets (cannot read secret values)'
+  static description = 'Fetch a list of set secrets (cannot read secret values)'
 
   static flags = {
-    help: flags.help({ char: 'h' }),
+    help: flags.help({char: 'h'}),
     personalAccessToken: flags.string({
       char: 't',
       description: 'Your GitHub Personal Access Token.',
@@ -18,19 +18,24 @@ export default class SecretsGet extends Command {
       description: 'Organisation the repo belongs to.',
       required: false,
     }),
-    repo: flags.string({ char: 'r', description: 'Name of the repo.', required: false }),
+    repo: flags.string({
+      char: 'r',
+      description: 'Name of the repo.',
+      required: false,
+    }),
   }
 
   async run() {
-    const { flags } = this.parse(SecretsGet)
+    const {flags} = this.parse(SecretsGet)
 
     try {
       const conf = await configuration(this)
 
       const requestWithAuth = request.defaults({
         headers: {
-          authorization: `token ${flags.personalAccessToken ?? conf.personalAccessToken
-            }`,
+          authorization: `token ${
+            flags.personalAccessToken ?? conf.personalAccessToken
+          }`,
         },
       })
       const result = await requestWithAuth(

--- a/src/commands/secrets/set.ts
+++ b/src/commands/secrets/set.ts
@@ -1,16 +1,16 @@
-import { Command, flags } from '@oclif/command'
-import { CLIError } from '@oclif/errors'
-import { request } from '@octokit/request'
+import {Command, flags} from '@oclif/command'
+import {CLIError} from '@oclif/errors'
+import {request} from '@octokit/request'
 import cli from 'cli-ux'
 import fs from 'fs-extra'
 import sodium from 'tweetsodium'
-import { configuration } from '../../utils/config'
+import {configuration} from '../../utils/config'
 
 export default class SecretsSet extends Command {
   static description = 'Update/Create a secret'
 
   static flags = {
-    help: flags.help({ char: 'h' }),
+    help: flags.help({char: 'h'}),
     personalAccessToken: flags.string({
       char: 't',
       description: 'Your GitHub Personal Access Token.',
@@ -21,7 +21,11 @@ export default class SecretsSet extends Command {
       description: 'Organisation the repo belongs to.',
       required: false,
     }),
-    repo: flags.string({ char: 'r', description: 'Name of the repo.', required: false }),
+    repo: flags.string({
+      char: 'r',
+      description: 'Name of the repo.',
+      required: false,
+    }),
     file: flags.string({
       char: 'f',
       description: 'Location of a file to create a secret from',
@@ -46,7 +50,7 @@ export default class SecretsSet extends Command {
   }
 
   async run() {
-    const { flags } = this.parse(SecretsSet)
+    const {flags} = this.parse(SecretsSet)
 
     try {
       const conf = await configuration(this)
@@ -55,11 +59,11 @@ export default class SecretsSet extends Command {
         headers: {
           authorization: `token ${
             flags.personalAccessToken ?? conf.personalAccessToken
-            }`,
+          }`,
         },
       })
 
-      const { data: token } = await requestWithAuth(
+      const {data: token} = await requestWithAuth(
         'GET /repos/{owner}/{repo}/actions/secrets/public-key',
         {
           owner: flags.org ?? conf.org,
@@ -108,12 +112,11 @@ export default class SecretsSet extends Command {
         )
         this.log('secret updated')
       } catch (error) {
-        this.log(error)
+        this.error(error)
         this.log('unable to update secret')
       }
     } catch (error) {
-      this.error(new CLIError(error))
-      this.exit(1)
+      this.error(new CLIError(error), {exit: 1})
     }
   }
 }

--- a/src/utils/config/index.ts
+++ b/src/utils/config/index.ts
@@ -1,8 +1,8 @@
 import chalk from 'chalk'
 import path from 'path'
 import fs from 'fs-extra'
-import { Command } from '@oclif/command'
-import { CLIError } from '@oclif/errors'
+import {Command} from '@oclif/command'
+import {CLIError} from '@oclif/errors'
 
 interface Config {
   org: string
@@ -22,7 +22,7 @@ export async function configuration(ctx: Command) {
       })
     }
 
-    const { org, repo, personalAccessToken }: Config = await fs.readJson(config)
+    const {org, repo, personalAccessToken}: Config = await fs.readJson(config)
 
     if (!org || !repo || !personalAccessToken) {
       ctx.warn(

